packages/gatsby-theme-carbon/package.json

```diff
@@ -33,8 +33,8 @@
     "@reach/router": "^1.2.1",
     "@vimeo/player": "^2.9.1",
     "beautiful-react-hooks": "^0.23.1",
-    "carbon-components": "^10.23.2",
-    "carbon-components-react": "^7.23.2",
+    "carbon-components": "^10.30.0",
```

Comment:
I'm not sure if `Dropdown` is used in the theme, but I know `10.30.0` has that chevron bug, so we might want to just bump this to `10.31.0`